exception_reports/decorators.py

```diff
@@ -41,10 +41,20 @@ def _exception_reports(func, *args, **kwargs):
             filename = gen_error_filename(extension='html')
             report_location = storage_backend.write(filename, data)
             setattr(e, 'report', report_location)
+
+            ExceptionType = type(e)
+
+            def my_str(self):
+                m = ExceptionType.__str__(self)
+                return f'{m} [report:{report_location}]'
+
+            NewExceptionType = type(ExceptionType.__name__, (ExceptionType,), {'__str__': my_str})
```

Comment:
```python
__dict__ = dict(ExceptionType.__dict__)
__dict__['__str__'] = my_str
NewExceptionType = type(ExceptionType.__name__, ExceptionType.__bases__, __dict__)
```
^ if you want the two monkey class version to be identical other than the replaced `__str__`, I believe you would have to do this. :)

Reply:
ok I added tests and I feel comfortable with this